<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Broker</class>
 <widget class="QMainWindow" name="Broker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frmInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>201</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="lblCompanyName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Название:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lnlINN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>52</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ИНН:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblINNTxt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>52</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCompanyNameTxt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>32</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUserChanges">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>72</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить пароль</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUserInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Данные о брокере:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAppendix">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>118</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Дополнительно:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAccounts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>138</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Открытых счетов:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAccountTxt">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblCheckTxt">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>158</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblCkecks">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>156</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Доступных чеков:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>194</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Меню</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAccountDel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>247</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Закрыть счёт</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAccountAppend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>224</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Зарегистрировать счёт</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReports">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>292</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть чеки (отчёты)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBudgetOper">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>269</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить баланс</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmOutput">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>591</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnOperations">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Операции</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAccounts">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>0</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Счета</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStocks">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>0</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ценные бумаги</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tblOutput">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>591</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmSearch">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="lblSearch">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Выберите инвестора (по ИНН):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>-1</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QFrame" name="frmAccAppend">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnAccept">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>ОК</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtFIO">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>Иванов C.C.</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblINNInv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ИНН:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFIO">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Фамилия И.О.:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtINNInv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::InputMethodHint::ImhNone</set>
     </property>
     <property name="inputMask">
      <string comment="000111222333"/>
     </property>
     <property name="text">
      <string>000111222333</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmBudget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Снять</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Положить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Укажите сумму в рублях</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
